### PR TITLE
Add chrony.conf change RHCOS to 4.8 Release Notes

### PR DESCRIPTION
**Pull request by bobfuru**
Request in 4.8 Release Notes tracker to add this note: https://github.com/openshift/openshift-docs/issues/29652#issuecomment-841302131

@travier and @miabbott PTAL for technical and QE review. Thanks!

**PREVIEW LINK:** https://deploy-preview-33242--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-rhcos-chrony-chrony-default

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-chrony-chrony-default"]
+==== Change to custom `chrony.conf` default on cloud platforms
+
+If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default.
```
Comment (miabbott) [inline release_notes/ocp-4-8-release-notes.adoc:51]: Maybe worth linking to the BZ that the upstream PR addressed - https://bugzilla.redhat.com/show_bug.cgi?id=1924869

Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc]: Added BZ, thanks.

**Comment by bobfuru** [comment]
@miabbott and @travier - does this lgty now?

**Review by travier — APPROVED** [review]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-chrony-chrony-default"]
```
Comment (lbarbeevargas) [inline release_notes/ocp-4-8-release-notes.adoc:48]: Tiny thing: do you need the double `chrony` in the id?

**Comment by lbarbeevargas** [comment]
One small nit, but otherwise LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc:48]: Whoa, nope! Thanks!

**Comment by bobfuru** [comment]
TL;DR: I'm closing this PR and following it up with https://github.com/openshift/openshift-docs/pull/33368. 

I had to resolve a rebase conflict in this PR, which added a second commit. Squashing that second commit (13f9698) was unsuccessful, possibly because there were no changes in it. Rather than merging this PR with 2 commits, I created 33368 to merge the chrony.conf RHCOS release note with a single commit per our doc guidelines.

**Comment by vikram-redhat** [comment]
@bobfuru reminder to remove milestone and branch label if you close a PR without merging it.
